forward_command_controller/src/forward_command_controller.cpp

```diff
@@ -34,31 +34,46 @@ ForwardCommandController::ForwardCommandController()
   joints_command_subscriber_(nullptr)
 {}
 
+controller_interface::return_type
+ForwardCommandController::init(
+  const std::string & controller_name)
+{
+  auto ret = ControllerInterface::init(controller_name);
+  if (ret != controller_interface::return_type::SUCCESS) {
+    return ret;
+  }
+
+  try {
```

Comment:
Do we have to declare Paramus in the Init step, or can this be done in the configure step? (If in configure, maybe it doesn’t work for PAL use-case)

Reply:
All PAL ported controllers declared them in init and I'd like to think I'm familiar with their use case. It should be fine as-is